librz/core/cmd/cmd_alias.c

```diff
@@ -0,0 +1,159 @@
+// SPDX-FileCopyrightText: 2024 RizinOrg <info@rizin.re>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "rz_cmd.h"
+#include "rz_core.h"
+
+static int rz_cmd_alias(void *data, const char *input) {
```

Comment:
maybe add a comment here as `TODO` where you explain that this needs to be refactored.

Reply:
ok 707f3464e6d499a09fc307547a37029557860b9b